package.json

```diff
@@ -13,9 +13,12 @@
     "blessed": "^0.1.81",
     "blessed-contrib": "^4.7.5",
     "chalk": "^1.1.3",
-    "git-standup": "^2.1.8",
+    "git-user-name": "^1.2.0",
+    "gitlog": "^2.4.0",
+    "is-git": "0.0.1",
     "parrotsay-api": "^0.1.1",
     "scraperjs": "^1.2.0",
+    "subdirs": "github:mojoaxel/subdirs",
```

Comment:
This is not perfect. I'm waiting for https://github.com/jarofghosts/subdirs/pull/2 to get merged...